fix(xo-web/proxies): remove "Bind license" for proxies without VM UUID (#6472)
vatesfr/xen-orchestra: CHANGELOG.unreleased.md

- [Plugin/auth-saml] Certificate input support multiline (PR [#6403](https://github.com/vatesfr/xen-orchestra/pull/6403))
- [Backup] Launch Health Check after a full backup (PR [#6401](https://github.com/vatesfr/xen-orchestra/pull/6401))
- [Backup] Fix `Lock file is already being held` error when deleting a VM backup while the VM is currently being backed up

> - [Licenses] Disable "Bind license" button for proxies without VM UUID (PR [#6472](https://github.com/vatesfr/xen-orchestra/pull/6472))

Reviewer: Users don't necessarily know technical details like the fact that we use the VM UUID to bind it to a proxy:
```suggestion
- [Licenses] Disable "Bind license" button for proxies whose corresponding VM cannot be found (PR [#6472](https://github.com/vatesfr/xen-orchestra/pull/6472))
```
Don't forget to change "Disable" into "Remove" if you agree with my comment about removing the button entirely.
